### DISCUSSION TITLE
Add new workflow on project about link issue with PR

### DISCUSSION THREAD

**Discussion by M97Chahboun** [community/community#65026] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I want suggest to add new workflow about link issue with pull request = move issue to in progress

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ExEr7um** [comment]
I want something similar. When I open pull request and link it with issue, I want the issue to go to "In review" status.

**Comment by M97Chahboun** [comment]
We need new workflow about link issue with PR and make it dynamic for add status by user

**Comment by TaylorHere** [comment]
when link issue or PR set item's status
